es-app/src/MetaData.cpp

```diff
@@ -19,6 +19,7 @@ MetaDataDecl gameDecls[] = {
 	{"publisher",	MD_STRING,				"unknown",			false,		"publisher",			"enter game publisher"},
```

Comment:
Happy to use space indentation for laying out stuff like this (but keep the tabs for now for the general code indentation).